chromium_src/chrome/browser/ui/bookmarks/bookmark_tab_helper.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/pref_names.h"
+
+#define BRAVE_SHOULD_SHOW_BOOKMARK_BAR \
```

Comment:
We can do this by subclassing with copying the base class `ShouldShowBookmarkBar()` method.
To avoid copying, this overriding and this patching is used.